include/rocksdb/statistics.h

```diff
@@ -411,6 +411,9 @@ struct HistogramData {
   double percentile99;
   double average;
   double standard_deviation;
+  // zero-initialize new members since old Statistics::histogramData()
```

Comment:
if a user has an old subclass of Statistics, their implementation of histogramData() won't write anything to max until they update it. I updated ours via the change in util/histogram.cc, which is used by StatisticsImpl::histogramData()